packages/frontend/src/contexts/unirep.js

```diff
@@ -259,4 +262,41 @@ export default class Unirep {
     this.signUpCount = data.data.users.length
     this.attesterCount = data.data.attesters.length
   }
+
+  async updateAttesterDescription(
+    attesterId,
+    network,
+    icon,
+    url,
+    name,
+    description,
+    signature,
+    nonce
+  ) {
+    const data = await fetch(`${SERVER}/api/about/${attesterId}`, {
+      method: 'POST',
+      headers: {
+        'content-type': 'application/json',
+        network: network,
```

Comment:
currently, network is of the form `arbitrum-goerli` but the backend requires `ArbitrumGoerli`. Either we can change the backend to use kebab case too or convert kebab case to PascalCase using 
```js
network = network.replace(/(^\w|-\w)/g, (text) =>
      text.replace(/-/, '').toUpperCase()
    )
    ```

Reply:
It can just be either `arbitrum-goerli` or `ArbitrumGoerli`
we can even get network information from backend
to be consistent with subgraph api, it could be `arbitrum-goerli`
https://api.studio.thegraph.com/query/48080/arbitrum-goerli/v2.0.0-beta-3

Reply:
I mean to maintain the network list in backend
https://github.com/Unirep/explorer/blob/4d87ec2cf7db4f64b5c100e64a10e0b422299ae5/packages/frontend/src/contexts/utils.js#L3